src/org/exist/client/InteractiveClient.java

```diff
@@ -1309,18 +1346,15 @@ private synchronized boolean findRecursive(final Collection collection, final Pa
     /**
      * stores given Resource
      *
-     * @param fileName simple file or directory
+     * @param file file or directory
      * @throws XMLDBException
      */
-    protected synchronized boolean parse(String fileName) throws XMLDBException {
+    protected synchronized boolean parse(final Path file) throws XMLDBException {
```

Comment:
@shabanovd I have been trying to tell you that this isn't a problem. Only Windows needs to handle Windows file paths!